open-company/open-company-web
# Digest links fixes

Review with: https://github.com/open-company/open-company-auth/pull/87

Bug 1: open a digest link (with the id-token), check the add comment field: medium-editor is never initialized since it waits for the roster users to do so.
Bug 2: open a digest link for a post that was removed: you never get the 404 page, the loading carrot is always there.

To test:
- add a post
- send yourself a digest
- open the post link in incognito
- [x] do you get MediumEditor in the add comment field?
- [x] do you see the roster call in the Network tab?
- [x] can you mention other users?
- now open the same link in a logged in session
- [x] do you get MediumEditor in the add-comment field?
- [x] can you mention a user?
- now copy the secure link of that post
- open the secure link in an incognito window
- [x] do you see the post?
- [x] no comments or add comment field?
- [x] everything works?
- now delete the post you were testing
- open the digest link in incognito
- [x] do you get a 404?
- open the digest link in a logged in session
- [x] do you get a 404 or post removed banner?
